cmd/cli/install.go

```diff
@@ -290,9 +293,24 @@ func (i *installCmd) validateOptions() error {
 		}
 	}
 
+	//validate the envoy log level type
+	if err := isValidEnvoyLogLevel(i.envoyLogLevel); err != nil {
+		return err
+	}
+
 	return nil
 }
 
+func isValidEnvoyLogLevel(envoyLogLevel string) error {
+	allowedLogLevels := []string{"trace", "debug", "info", "warning", "warn", "error", "critical", "off"}
```

Comment:
Good stuff! Thanks for adding this!
It would be good to put the link to the Envoy source code in a comment above this:  https://github.com/envoyproxy/envoy/blob/release/v1.15/test/server/options_impl_test.cc#L373  (for example)

Reply:
Maybe omit the line number, because that's going to keep changing.

Reply:
@shashankram that's why I put the `release/v1.15` in the link - should be a canonical link to a commit.

Reply:
I've added the reference, @delqn PTAL